### DISCUSSION TITLE
Tradução da nova documentação.

### DISCUSSION THREAD

**Discussion by eduardomoroni** [reactjs/pt-br.react.dev#553] [Ideas]
Olá @reactjs/reactjs-org-portuguese-brazil-translation, a documentação beta está no [AR](https://beta.reactjs.org/). 
Já criaram um issue sobre o assunto, https://github.com/reactjs/reactjs.org/issues/4135

Como vocês acham que devemos proceder? Parar completamente a tradução da documentação atual e focar na beta?
Tinha várias pessoas que queriam ajudar com a tradução da documentação e num tinha página suficiente para a quantidade de pessoas, como podemos chamar esse pessoal?

**Comment by joemaffei** [comment]
Meu Português está um pouco enferrujado esses dias, mas eu gostaria de ajudar com a tradução. Concordo que deveríamos focar na beta, mas como a documentação atual ainda está "viva", acho que também deveria ser mantida. A tradução atual foi completada, ou ainda falta conteúdo para traduzir?

**Comment by eduardomoroni** [comment]
Se não me engano, o que faltava da documentação atual eram os blog post.
Porém, acredito que o foco deva ser dar um gás na beta, pois em breve vai
ser substituída.
Não se preocupe muito não em ter um português perfeito, vai ter pessoas
npara revisar e o mais importante inicialmente é ter algo entendível.

**Reply by iaurg** [reply]
Como comentou que faltam apenas os posts do blog creio que dê para manter ambos, pois provavelmente posts sejam migrados ou continuem sem alteração, pois são conteúdos informativos não relacionados com a documentação em sí. Eles comentam em algum lugar sobre a migração desse conteúdo existente no blog?

https://beta.reactjs.org/blog

https://reactjs.org/blog/

**Reply by eduardomoroni** [reply]
Não estou tão a par da situação assim, mas creio que pode ser aproveitado. O que comentaram por agora é começar a tradução da pasta https://github.com/reactjs/pt-br.reactjs.org/tree/main/beta

**Comment by joemaffei** [comment]
O que precisamos fazer para trazer a documentação beta para este repositório para começar a tradução? Vejo dois pull requests abertos que eu suponho contêm a nova documentação.

**Reply by eduardomoroni** [reply]
Criei um issue para manter atualizado quem ta fazendo o que e como ta indo https://github.com/reactjs/pt-br.reactjs.org/issues/555

**Comment by cezaraugusto** [comment]
@eduardomoroni acho valido parar a atual e focar no beta. Pode criar um checkbox em markdown para cada página que você listou na #555. Depois pegar a lista de contribuidores em https://github.com/reactjs/pt-br.reactjs.org/issues/1 e fazer uma chamada talvez?

**Comment by eduardomoroni** [comment]
Posso sim, amanhã eu faço. Se você quiser/puder edita a minha mensagem lá.
Sem problemas

**Comment by jhonmike** [comment]
Bom dia pessoal, dei uma revisada e "mergiada" nos PRs que estavam abertos, também aproveitei para atualizar o PR de sync do bot da doc em Inglês!! vou ficar nesse primeiro momento mais focado nas reviews por causa do pouco tempo!! mas qualquer coisa é só me marcar!!

Feliz ano novo para todos!!

**Comment by jhonmike** [comment]
Bom noite para quem esta no Brasil, pessoal agora sim, site no ar e todos os conflitos tratados, bora restartar as traduções, começei pela pagina inicial hoje e aos poucos vou atualizando nossa issue de tradução!! Let's Bora!!!

**Comment by renato-bohler** [comment]
Trazendo [a discussão](https://github.com/reactjs/pt-br.react.dev/issues/555#issuecomment-1528807675) pra cá.

P.S.: acho que seria benéfico ter alguém pra definir como traduzir estes termos ao invés de colocarmos tudo pra discussão/voto, porque consigo enxergar essas discussões nunca tendo fim (afinal é tudo bastante subjetivo 😅). Mesmo que seja escolhida uma tradução que não seja ideal, acredito que a consistência na tradução da documentação como um todo seja extremamente importante: mais confuso do que ter um termo confuso é ter vários termos confusos diferentes em cada página.

P.S. do P.S.: talvez poderíamos criar uma discussão separada pra essas questões também.

---

Alguma recomendação em específico em como traduzir _"top level"_? Seria legal ter algo no glossário sobre isso, já que esse termo é usado em todas as documentações de hooks.

Por exemplo, um trecho do texto em inglês:

> Call `useEffect` at the top level of your component to declare an Effect

Opções que enxergo:

---
> Execute `useEffect` na raiz de seu componente para declarar um Effect

(a minha escolha - soa mais como português nativo, parece razoavelmente fácil de compreender)

---

> Execute `useEffect` no top level de seu componente para declarar um Effect

(me parece mais fácil de entender pra quem entende um pouco das terminologias)

---

> Execute `useEffect` no nível superior de seu componente para declarar um Effect

("nível superior" me soa estranho 🤔)

**Reply by TMendes-lucca** [reply]
Acredito que raiz soe melhor também, mesmo pensando em alguém que está começando agora, como o termo raiz referência o próprio componente, não vejo uma forma ambigua de interpretar a questão. O termo 'nível superior' também me soa estranho

**Comment by augustodg2** [comment]
Algum padrão para traduzir input no sentido de campo de formulário? Não encontrei no glossário

**Reply by jhonmike** [reply]
eu deixaria input mesmo, podemos colocar no Glossary se mais pessoas concordarem

**Comment by ojuliocode** [comment]
Qual o estado da tradução atual ?Há contribuintes ativos?

**Reply by jhonmike** [reply]
olá @ojuliocode não vou tiver 100% ativos mas estamos aii, tem alguns PRs para review abertos já e alguns para traduzir

**Reply by jhonmike** [reply]
se vc quiser traduzir pode olhar essa issue aqui https://github.com/reactjs/pt-br.react.dev/issues/555
e se quiser revisar algum PR aberto pode olhar PRs de contribuites aqui https://github.com/reactjs/pt-br.react.dev/pulls

**Comment by DaveSimoes** [comment]
Dears. 
  Gostaria de contribuir com a tradução da seção:
 'startTransition' (src/content/reference/react/startTransition.md).
  Agora estou compreendendo melhor todos os  procedimentos e boas práticas para participação deste projeto.

Obrigado pela oportunidade !!
